vmcnet/models/construct.py

```diff
@@ -540,6 +552,16 @@ def get_residual_blocks_for_ferminet_backflow(
             to the two-electron stream with an optional skip connection, otherwise
             the two-electron stream is mixed into the one-electron stream but no
             transformation is done.
+        num_heads (int): number of heads for the transformer. The future todo is to
```

Comment:
Nit: prefer to record to-do like
`TODO (JiahaoYao): ...` 
so we can search codebase for todo's in a coherent way and so we can know who put the TODO in.

Reply:
good idea